changes/2819.chore.rst

```diff
@@ -0,0 +1 @@
+Ensure that invocations of ``create_array`` use consistent keyword arguments.
```

Comment:
I think it would be helpful for users to add more concretely what has changed, ie which functions have gained which arguments.

Reply:
https://github.com/zarr-developers/zarr-python/pull/2819/commits/3fa6526511200c25256c0c9abc5afac95803ff47